scapy/layers/inet.py

```diff
@@ -806,6 +806,15 @@ def mysummary(self):
         else:
             return self.sprintf("TCP %TCP.sport% > %TCP.dport% %TCP.flags%")
 
+    def guess_payload_class(self, payload):
```

Comment:
I think its not a good idea to import your contrib layer in guess_payload_class of TCP. Can you maybe move this port to dispatch_hook?